sdk/keyvault/azure-security-keyvault-secrets/CMakeLists.txt

```diff
@@ -117,7 +117,7 @@ if (BUILD_PERFORMANCE_TESTS)
   add_subdirectory(test/perf)
 endif()
 
-if(BUILD_SAMPLES)
+if(BUILD_SAMPLES_DISABLED)
```

Comment:
Rather than changing the CMakeLists.txt, can we update the ci leg that's passing in `BUILD_SAMPLES=ON` in the yml, and we can revert that after?

Reply:
i'd rather avoid messing with yml files, also i don't want to disable globally for everyone